lib/Listener/LoadContactsFilesActions.php

```diff
@@ -39,5 +39,10 @@ public function handle(Event $event): void {
 		}
 
 		Util::addScript(Application::APP_ID, 'contacts-files-action');
+		// only available since Nc 28
+		if (method_exists(Util::class, 'addInitScript')) {
+			Util::addInitScript(Application::APP_ID, 'init');
+		}
```

Comment:
```suggestion
		// only available since Nc 28
		if (method_exists(Util::class, 'addInitScript')) {
			Util::addInitScript(Application::APP_ID, 'contacts-files-action');
		} else {
			Util::addScript(Application::APP_ID, 'contacts-files-action');
		}
```

Reply:
This way you can even delete of init.ts as the existing bundle will be loaded in both cases. The conditional logic is already present in `files-action.js`.